### PR TITLE
Applied the Suggested Changes and Resolved #221

### LINKED ISSUE #221: Move personality type setting into the notification settings section
## Feature Reasoning

The personality configuration setting affects the personality type for chat and computer notifications. As such, it should be grouped with those settings in the configuration menu.

## Feature Description

As seen in the screenshot below, the personality type configuration setting appears at the bottom of the configuration menu. This enhancement is to place that setting into the `Notification Settings` section under `Computer notification`.

## Possible Alternatives

This setting may be moved in the future based on the addition of other configuration settings but for now, placing it under `Notification Settings` makes the most sense.

## Additional Context

Here is a screenshot of the current configuration menu:
image

To implement this enhancement, look into the configuration menu settings implemented within [HydrateReminderConfig.java](https://github.com/jmakhack/hydrate-reminder/blob/master/src/main/java/com/hydratereminder/HydrateReminderConfig.java).

Please refer to [CONTRIBUTING.md](https://github.com/jmakhack/hydrate-reminder/blob/master/CONTRIBUTING.md) for info on how to setup, build, run, and test this project. Feel free to ping @jmakhack in the comments below with any questions if needed. Also, join the [Discord](https://discord.gg/RTgxfFW9mS) to meet and engage with other contributors!

[](https://discord.gg/RTgxfFW9mS)

### PR DESCRIPTION
**Pull request by SaranshBaniyal**
## Associated Issue #221

### CODE REVIEW

**Comment by SaranshBaniyal** [comment]
@jmakhack I have created this new PR with all the suggested changes, please have a look

File: src/main/java/com/hydratereminder/HydrateReminderConfig.java [hunk 1]
```diff
@@ -151,6 +151,24 @@ default boolean hydrateReminderChatMessageEnabled()
...
+			name = "Personality type",
+			description = "Sets the type of personality by which different messages will be displayed when hydrating",
+			section = HYDRATE_REMINDER_NOTIFICATIONS_SECTION,
+			position = +1
+	)
+	default HydrateReminderPersonalityType hydrateReminderPersonalityType()
+	{
+		return HydrateReminderPersonalityType.FUN;
+	}
+
```
Comment (jmakhack) [inline src/main/java/com/hydratereminder/HydrateReminderConfig.java:171]: @SaranshBaniyal As mentioned in #221, the position of this setting should be below the `Computer notification` setting.

image

So two things need to be changed here:

1. The value of `position` should be 1 greater than the position of `Computer notification`
2. The placement of this config item code should be placed under the `Computer notification` config item code

**Review by jmakhack — CHANGES_REQUESTED** [review]
@SaranshBaniyal Please address the one comment I have on your changes. Also, there is no need to close this pull request and open a new one. If you push new code onto your branch, the pull request will be automatically updated.

File: src/main/java/com/hydratereminder/HydrateReminderConfig.java (hunk 1 above)
Comment (SaranshBaniyal) [reply src/main/java/com/hydratereminder/HydrateReminderConfig.java]: @jmakhack sure. I have pushed another commit, please have a look

**Review by jmakhack — APPROVED** [review]
The changes look good @SaranshBaniyal 

Thanks for the contribution! :)

**Comment by jmakhack** [comment]
@all-contributors please add @SaranshBaniyal for code

**Pull request merged**